src/storage-controller/src/lib.rs

```diff
@@ -2185,7 +2185,7 @@ where
         //   be called shortly).
         // - That all txn writes through `init_ts` have been applied
         //   (materialized physically in the data shards).
-        let removed = txns
+        let (removed, _tidy) = txns
```

Comment:
Not sure what to do with this `_tidy`, I think it's fine to ignore it and just let the next applier tidy the txn shard.

Reply:
Sgtm, I'd leave a comment with that info though

Reply:
Done.